src/components/security-rules/configure-rule/ConfigureRule.jsx

```diff
@@ -488,12 +489,15 @@ const ConfigureRule = (props) => {
         </ConditionalFormBlock>
         <ConditionalFormBlock
           dependency="rule"
-          condition={() => form.getFieldValue('rule') === "webhook"}
-        >
+          condition={() => form.getFieldValue('rule') === "webhook"} >
           <FormItemLabel name="URL" />
           <FormItem name="url" rules={[{ required: true }]}>
             <Input placeholder="URL" />
           </FormItem>
+          <FormItemLabel name="STORE" />
```

Comment:
Let's rename the label - `STORE` to `Store` in both the places. We only use all caps if it's an abbreviation (for e.g. URL).